src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,7 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="SynchronizingWithClassView" xml:space="preserve">
     <value>Synchronizing with {0}...</value>
   </data>
+  <data name="FullSolutionAnalysisOff" xml:space="preserve">
+    <value>Full solution analysis disabled due to low memory. go to tools-&gt;options and search "Enable full solution analysis" to turn it back on or restart VS to restore full solution analysis.</value>
```

Comment:
Yeah.  That's super-ick.  How about:

"Low memory detected.  Full solution analysis disabled."

--

also, if we want FullSolutionAnalysis to come back when this solution is closed and another is opened... why not just do that automatically?  We know when that has happened, so we could just automatically do that on behalf of the user.  Then we don't need any buttons.  We're just telling them what has happened.  And once the situation is cleared up, VS is working properly again.